src/implementations/vanilla/components/datepicker/datepicker.js

```diff
@@ -87,11 +84,16 @@ export class Datepicker {
    * Initialise component.
    */
   init() {
+    if (typeof window.Pikaday === 'undefined') {
```

Comment:
shouldn't we exit this function if pikaday is not available?
Otherwise it will still break the ecl js

Reply:
there is a `throw new TypeError` inside this if ;)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw